plugins/gpio/meson.build

```diff
@@ -1,9 +1,8 @@
-if get_option('plugin_gpio')
+gpio_header = cc.has_header_symbol('linux/gpio.h', 'GPIO_V2_LINE_FLAG_OUTPUT', required: get_option('plugin_gpio'))
+if get_option('plugin_gpio').enabled() or \
+   (get_option('plugin_gpio').auto() and gpio_header)
```

Comment:
Isn't the point of `required: get_option('plugin_gpio')` to make this *not* need manual checking of tristate values?

- if enabled: then has_header_symbol must have returned true, or the build would error
- if auto, then has_header_symbol will return true if found, and false if not found
- if disabled, then has_header_symbol immediately returns false without checking

So the only thing that matters is whether `if gpio_header`.

Reply:
Yeah, this was heavy handed.  I fixed it up, but I still need to detangle gudev still.